README.md

```diff
@@ -1,7 +1,10 @@
 jBBCode
 =======
+[![GitHub release](https://img.shields.io/github/release/jbowens/jBBCode.svg)](https://github.com/jbowens/jBBCode/releases)
```

Comment:
I'll keep this one but the other two don't seem necessary. I'd like to avoid too much clutter in the readme.